go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/chromedp/cdproto v0.0.0-20191114225735-6626966fbae4
 	github.com/chromedp/chromedp v0.5.2
 	golang.org/x/image v0.0.0-20210628002857-a66eb6448b8d
-	golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
-	golang.org/x/text v0.3.7
-	golang.org/x/tools v0.1.0
+	golang.org/x/sync v0.1.0
+	golang.org/x/text v0.11.0
+	golang.org/x/tools v0.6.0
```

Comment:
Are these required for this change? If so, can they be done in a separate change?